tensorpack/utils/utils.py

```diff
@@ -214,7 +214,9 @@ def get_tqdm_kwargs(**kwargs):
     return default
 
 
-def get_tqdm(**kwargs):
+def get_tqdm(df=None, **kwargs):
```

Comment:
This is not needed. `get_tqdm(iterable=df)` is the same.

Reply:
Or maybe add `*args, **kwargs`.

Reply:
`*args, **kwargs` looks good!